### PR TITLE
Merge rules using OneTrust and Didomi CMPs

### PR DESCRIPTION
**Pull request by Softvision-ValentinBandac**
Merged rules for Didomi:
jutarnji.hr, orange.fr, vecernji.hr, sport.es, elespanol.com, in-pocasi.cz, 20minutes.fr, actu.fr, hbvl.be, naszemiasto.pl with rule af4c5b38-d210-472b-9a07-21cbe53c85ab;  krone.at, cas.sk, nova.cz, heureka.sk, free.fr, markiza.sk, willhaben.at, francetvinfo.fr with rule 690aa076-4a8b-48ec-b52c-1443d44ff008; 
Merged rules for OneTrust:
rte.ie with rule 6c7366a0-4762-47b9-8eeb-04e86cc7a0c; tv2.dk with rule 0b42c238-b54d-4106-8d9a-81df5bc0b3ae; glassdoor.com with rule c4b8af6b-5e97-4214-badd-67f716d3e5b6; mailchimp.com with rule a3b0e77d-a7fe-4f38-8015-e368c67ecbbf; coursera.org with rule 8809db65-96cd-4ab5-be1a-cea416a4f34f.

### CODE REVIEW

File: cookie-banner-rules-list.json [hunk 1]
```diff
@@ -5996,19 +5934,6 @@
...
       "domains": ["healthline.com"]
     },
-    {
-      "click": {
-        "optIn": "button#onetrust-accept-btn-handler",
-        "optOut": "button#onetrust-reject-all-handler",
-        "presence": "div#onetrust-consent-sdk"
-      },
-      "cookies": {
-        "optIn": [{ "name": "_tt_enable_cookie", "value": "1" }],
```
Comment (Trikolon) [inline cookie-banner-rules-list.json:6006]: Same thing here, can't this just be merged with the main onetrust rule?

**Review by Trikolon — CHANGES_REQUESTED** [review]
Thanks! This already cleans up our list partially. However I think we should try to merge as many onetrust and didomi rules as possible.

Looking for `didomi-notice-agree-button` and `onetrust-consent-sdk` I get a lot of hits. Unless the banners are significantly different within the same CMP, can't these (mostly?) be merged?

File: cookie-banner-rules-list.json (hunk 1 above)
Comment (Softvision-ValentinBandac) [reply cookie-banner-rules-list.json:6006]: Tested the cookies and merged the rules accordingly.

File: cookie-banner-rules-list.json [hunk 2]
```diff
@@ -6171,19 +6096,6 @@
...
     },
-    {
-      "click": {
-        "optIn": "button#onetrust-reject-all-handler",
-        "optOut": "button#onetrust-reject-all-handler",
-        "presence": "div#onetrust-consent-sdk"
-      },
-      "cookies": {
-        "optIn": [{ "name": "OneTrustWPCCPAGoogleOptOut", "value": "false" }],
-        "optOut": [{ "name": "OneTrustWPCCPAGoogleOptOut", "value": "true" }]
```
[the remark this answers is not in the document]
Comment (Softvision-ValentinBandac) [reply cookie-banner-rules-list.json:6182]: Tested the cookies and merged the rules accordingly.

**Review by Trikolon — APPROVED** [review]
Awesome cleanup 🔥
Thank you!

**Pull request merged**